src/command/helpModule.ts

```diff
@@ -40,6 +41,34 @@ export default class HelpModule extends Module {
             );
         }, initialEmbed);
 
-        await msg.channel.send(embed);
+        const fallbackContent = modules.reduce((message, module) => {
+            const commands = _commands.filter(
+                command => command.module === module
```

Comment:
Pretty verbose/long name for a pretty temporary value IMO; Up to you to decide if you wanna keep it like this.